x-pack/plugins/uptime/public/pages/settings.tsx

```diff
@@ -76,16 +79,42 @@ const getFieldErrors = (formFields: DynamicSettings | null): SettingsPageFieldEr
       heartbeatIndices: indError,
       expirationThresholdError: expError,
       ageThresholdError: ageError,
+      invalidEmail: hasInvalidEmail(formFields.defaultConnectors, formFields.defaultEmail),
     };
   }
   return null;
 };
 
+const hasInvalidEmail = (defaultConnectors: string[], value?: DefaultEmail) => {
```

Comment:
Might be some code reuse opportunities, as a lot of this logic is repeated in `default_email.tsx`